groups/infrastructure/variables.tf

```diff
@@ -0,0 +1,88 @@
+# Environment
+variable "environment" {
+  type        = string
+  description = "The environment name, defined in envrionments vars."
+}
+variable "aws_region" {
+  default     = "eu-west-2"
+  type        = string
+  description = "The AWS region for deployment."
+}
+variable "aws_profile" {
+  default     = "development-eu-west-2"
+  type        = string
+  description = "The AWS profile to use for deployment."
+}
+
+variable "cert_domain" {
+  type        = string
+  description = "The certificate domain to use."
+}
+
+# EC2
+variable "ec2_key_pair_name" {
+  type        = string
+  description = "The key pair for SSH access to ec2 instances in the clusters."
+}
+variable "ec2_instance_type" {
+  default     = "t3.medium"
+  type        = string
+  description = "The instance type for ec2 instances in the clusters."
+}
+variable "ec2_image_id" {
+  default     = "ami-04018f95156d810bc" # ECS optimized Amazon2 Linux in London created 15/03/2023
```

Comment:
This should not be hard coded, or this must be overridden by a pipeline-based `ami-id` resource.

Reply:
This is fixed now. This is removed & using the terraform module default. Commit is pushed.